LittleHelpBook/Dockerfile

```diff
@@ -19,4 +30,5 @@ RUN dotnet publish "LittleHelpBook.csproj" -c Release -o /app/publish
 FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
+
 ENTRYPOINT ["dotnet", "LittleHelpBook.dll"]
```

Comment:
I'm a stickler for ending files with a newline - that way you can `cat` them on the command line and it doesn't mess up your prompt.

Reply:
I never knew why linters enforce having a blank line at the end of a file. Now, I know, and it makes sense. I have put newlines at the ends of the files. PR #3